src/service/jank-meter.js

```diff
@@ -63,37 +69,59 @@ export class JankMeter {
     }
     const paintLatency = this.win_.Date.now() - this.scheduledTime_;
     this.scheduledTime_ = null;
-    this.totalCnt_++;
+    this.totalFrameCnt_++;
     if (paintLatency > 16) {
-      this.jankCnt_++;
+      this.badFrameCnt_++;
       dev().info('JANK', 'Paint latency: ' + paintLatency + 'ms');
     }
 
-    // Report Good Frame Probability on Nth frame.
-    if (this.perf_ && this.totalCnt_ == NTH_FRAME) {
-      this.perf_.tickDelta('gfp', this.calculateGfp_());
+    // Report metrics on Nth frame, so we have sort of normalized numbers.
+    if (this.perf_ && this.totalFrameCnt_ == NTH_FRAME) {
+      // gfp: Good Frame Probability
+      const gfp = this.calculateGfp_();
+      this.perf_.tickDelta('gfp', gfp);
+      // bf: Bad Frames
+      this.perf_.tickDelta('bf', this.badFrameCnt_);
+      if (this.longTaskObserver_) {
+        // lts: Long Tasks of Self frame
+        this.perf_.tickDelta('lts', this.longTaskSelf_);
+        // ltc: Long Tasks of Child frames
+        this.perf_.tickDelta('ltc', this.longTaskChild_);
+        this.longTaskObserver_.disconnect();
+        this.longTaskObserver_ = null;
+      }
+      let batteryDrop = 0;
```

Comment:
What's the scope of battery drop? Is it per frame? Per life of page? W/o some normalization, it's kind of hard to get good value out of it. Also, the battery drain depends on many things that happen on the phone: notably a background tasks for audio playback. Or even foreground video playback and such on the page itself.